keybert/_model.py

```diff
@@ -1,3 +1,5 @@
+# ruff: noqa: E402
```

Comment:
This was done in Bertopic as well.
Ref: https://github.com/MaartenGr/BERTopic/blob/bf1fedd5bb14bbf282d5051e849d2b06abaa8b51/bertopic/_bertopic.py#L1